aea/protocols/generator.py

```diff
@@ -278,52 +295,60 @@ def _message_class_str(self) -> str:
         )
         cls_str += "        try:\n"
 
-        cls_str += "            assert isinstance(\n"
-        cls_str += "                self.dialogue_reference, Tuple\n"
-        cls_str += (
-            "            ), \"dialogue_reference must be 'Tuple' but it is not.\"\n"
-        )
-        cls_str += "            assert isinstance(\n"
-        cls_str += "                self.dialogue_reference[0], str\n"
-        cls_str += "            ), \"The first element of dialogue_reference must be 'str' but it is not.\"\n"
-        cls_str += "            assert isinstance(\n"
-        cls_str += "                self.dialogue_reference[1], str\n"
-        cls_str += "            ), \"The second element of dialogue_reference must be 'str' but it is not.\"\n"
+        cls_str += '            assert type(self.dialogue_reference) == tuple, "dialogue_reference must be \'tuple\' but it is not."\n'
+        cls_str += '            assert type(self.dialogue_reference[0]) == str, "The first element of dialogue_reference must be \'str\' but it is not."\n'
+        cls_str += '            assert type(self.dialogue_reference[1]) == str, "The second element of dialogue_reference must be \'str\' but it is not."\n'
```

Comment:
@5A11 you don't need to use `\'` if you use `'` inside`""`, you can just do `" ...'str'..."`

Reply:
Done